File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/components/parts/FieldOption.java
package cz.cvut.fel.matyapav.afandroid.components.parts;

/**
 * @author Pavel Matyáš (matyapav@fel.cvut.cz).
 *
 *@since 1.0.0..
 */
public class FieldOption {

    private String key;
    private String value;

    public FieldOption() {
    }

    public FieldOption(String key, String value) {
        this.key = key;
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return "FieldOption{" +
                "key='" + key + '\'' +
                ", value='" + value + '\'' +
                '}';
    }
}

File: AFMobile/AFWindowsPhone/AFWindowsPhone/components/parts/FieldOption.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AFWindowsPhone.builders.components.parts
{
    class FieldOption
    {
        private String key;
        private String value;

        public FieldOption()
        {
        }

        public String getKey()
        {
            return this.key;
        }

        public void setKey(String key)
        {
            this.key = key;
        }

        public String getValue()
        {
            return this.value;
        }

        public void setValue(String value)
        {
            this.value = value;
        }

    }
}
